creative/renderers/display/renderer.js

```diff
@@ -1,6 +1,17 @@
+import { MESSAGE_EVENT } from '../../constants.js';
 import {ERROR_NO_AD} from './constants.js';
 
-export function render({ad, adUrl, width, height, instl}, {mkFrame}, win) {
+export function render({ad, adUrl, width, height, instl}, {mkFrame, sendMessage}, win) {
+  if (window.ReportingObserver) {
```

Comment:
`window` here is not the window where the creative will be; however, from my testing using the "correct" `win.ReportingObserver` doesn't work - so I guess this is OK.

This made me realize however that this likely won't work when the GAM creative is not set up to use safeframe (which is quite common). The logic there:

https://github.com/prebid/Prebid.js/blob/71139a19d731c8362cf3ac91c3a830feacaf71e1/src/adRendering.ts#L326-L344

doesn't run this renderer at all when the bid provides `ad`; and runs it using a different window setup otherwise. I'd test this by calling `pbjs.renderAd` directly (e.g. following https://github.com/prebid/Prebid.js/blob/master/integrationExamples/noadserver/basic_noadserver.html) - we probably need to add the `ReportingObserver` logic in core as well.